src/maintenance/repositories/index.md

```diff
@@ -0,0 +1,4 @@
+# Repositories
+
+Repositories are the heart of the xbps package system. Repositories can be
+locally or remotely available.
```

Comment:
Maybe define what a repository is in the sense of xbps here?  i.e. that it contains a -repodata file?